service-registry-client/src/main/java/no/difi/meldingsutveksling/serviceregistry/externalmodel/InfoRecord.java

```diff
@@ -0,0 +1,32 @@
+package no.difi.meldingsutveksling.serviceregistry.externalmodel;
+
+import java.io.Serializable;
+
+/**
+ *
+ */
+public class InfoRecord implements Serializable {
+    private final String primaryServiceIdentifier;
+    private final String organizationNumber;
+
+    public InfoRecord(String primaryServiceIdentifier, String organizationNumber) {
+        this.primaryServiceIdentifier = primaryServiceIdentifier;
+        this.organizationNumber = organizationNumber;
+    }
+
+    public String getPrimaryServiceIdentifier() {
+        return primaryServiceIdentifier;
+    }
+
+    public String getOrganizationNumber() {
+        return organizationNumber;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Foreslår at vi legger oss på MoreObjects.ToStringHelper (fra Guava) for enkel konsistens.